docs/quickstart.rst

```diff
@@ -142,7 +142,7 @@ While this page shows only screenshots, the actual dashboard is interactive.
 .. note::
 
     The :code:`FairlearnDashboard` will move from Fairlearn to the
-    :code:`raiwidgets` package after the v0.5.0 release. Instead, Fairlearn
+    :code:`raiwidgets` package in the near future. Instead, Fairlearn
```

Comment:
we can point users to the dashboard package already since it's released. We should also raise a warning when they use the dashboard and point them to the other package.

Reply:
In the code, we're doing that already:
https://github.com/fairlearn/fairlearn/blob/44244c6a513eb9b8446369d3c92dce4adf1e4f7e/fairlearn/widget/_fairlearn_dashboard.py#L47
Let me redraft these now that raiwidgets is published to PyPi